docs/02-guides/public-configuration.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Public configurations
```

Comment:
**issue:** title doesn't express an action (How-to xxxx)

**suggestion:** reword. Ideas:
- Use public configurations client-side
- Read and extend public configurations
- Expose configurations client-side
- Access configurations in React
- Access configurations client-side